components/serializer.rst

```diff
@@ -674,6 +674,63 @@ you indicate that you're expecting an array instead of a single object.
 
     $data = ...; // The serialized data from the previous example
     $persons = $serializer->deserialize($data, 'Acme\Person[]', 'json');
+	
+The ``XmlEncoder``
+------------------
+
+This encoder transforms arrays into XML and vice versa. For example, take an
+object normalized as following::
+
+    array('foo' => array(1, 2), 'bar' => true);
+
+The ``XmlEncoder`` encodes this object as follows:
+
+.. code-block:: xml
+
+    <?xml version="1.0"?>
+    <response>
+        <foo>1</foo>
+        <foo>2</foo>
+        <bar>1</bar>
+    </response>
+
+The array keys beginning with ``@`` are considered XML attributes::
+
+    $encoder = new XmlEncoder();
+    $encoder->encode(array('foo' => array('@bar' => 'value')));
+    // will return:
+    // <?xml version="1.0"?>
+    // <response>
+    //     <foo bar="value" />
+    // </response>
+	
+Context
+~~~~~~~
+
+The XmlEncoder ``encode()`` method defines a third optional parameter called
+``context`` to define some configuration options for the XmlEncoder::
+
+    $xmlEncoder->encode($array, 'xml', $context);
+
+These are the options available:
+	
+``xml_format_output``
+    If set to true, format the output XML with line breaks and indentation.
+
+``xml_version``
+    Change the XML version attribute.
+
+``xml_encoding``
+    Change the XML encoding attribute.
+
+``xml_standalone``
+    Add standalone attribute in XML output .
+
+``xml_root_node_name``
+    Change the root node name (default: ``response``).
+
+``remove_empty_tags``
```

Comment:
Context options related to the XML encoder should not be documented in the main chapter of the serializer doc IMO (but this should be decided by the doc team).
And if we document them (which is a good idea), context options supported by encoders for other formats should be documented too.